plugin/evm/config/config.go

```diff
@@ -27,6 +27,11 @@ type Config struct {
 	// default to use the parent block's target gas per second.
 	GasTarget *gas.Gas `json:"gas-target,omitempty"`
 
+	// MinDelayTarget is the minimum delay per second that this node will attempt to use
+	// when creating blocks. If this config is not specified, the node will
+	// default to use the parent block's target delay per second.
```

Comment:
I don't think this comment is quite accurate. This seems like the `minimum time between blocks denominated in milliseconds` rather than the `minimum delay per second`